includes/definitions/eltex-mes.yaml

```diff
@@ -0,0 +1,41 @@
+os: eltex-mes
+text: Eltex-mes
+ifname: true
+type: network
+icon: eltex
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.35265.1.112
+            - .1.3.6.1.4.1.35265.1.83
+
+mib_dir: eltex
+
+poller_modules:
```

Comment:
Please Remove all of these if not needed due to buggy device

Reply:
hi @Jellyfrog 
sorry, i am don't understand your request

mibs from mibs/eltex are used in
includes/definitions/discovery/eltex-mes.yaml 
to discover CPU,fans,battery charge and chassis temperature

for example
 oid: rlCpuUtilDuringLast5Minutes
 num_oid: '.1.3.6.1.4.1.89.1.9.{{ $index }}'

 oid: rlEnvMonFanStatusTable
 value: rlEnvMonFanState

Reply:
hi @Jellyfrog 

> Please Remove all of these if not needed due to buggy device

the OSPF & BGP is removed

and most important
bad_iftype:
    - tunnel
    - softwareLoopback
    - propVirtual
    - propPointToPointSerial

because eltex have initially dozen of loopback port, so they are removed
so, yes, i am think it is correct this way

Reply:
If the modules isn't causing any issues they shouldn't be disabled. Bad if types are fine